mods/persistence/modules/projectiles/guns/energy/tier1/lasrifle.dm

```diff
@@ -3,10 +3,16 @@
 	desc = "The Marine model is one of the more ubiquitous energy weapon designs. While its capacity leaves something to be desired, it is incredibly easy to mass-produce and has great beam strength for its low manufacturing cost. It is often seen in the hands of corporate space marines, hence the name."
 	icon = 'mods/persistence/icons/obj/guns/tier1/lasrifle.dmi'
 	slot_flags = SLOT_BACK
-	w_class = ITEM_SIZE_LARGE
+	w_class = ITEM_SIZE_HUGE
 	one_hand_penalty = 3
 	max_shots = 5
 	fire_delay = 12
 	force = 10
 	origin_tech = "{'combat':3,'magnets':2,'engineering':2,'materials':3}"
```

Comment:
```suggestion
	origin_tech = "{'combat':3,'magnets':2,'engineering':2,'materials':999}"
```
Try this instead of putting the required tech on the recipes. I misread how it was interpreted on recipes.

Reply:
This is for one of the guns I do want craftable. That said, do check the new updates, I did something similar.